torch/csrc/jit/codegen/cuda/lower_unroll.h

```diff
@@ -86,6 +91,8 @@ class TORCH_CUDA_API UnrollPass {
   // Map from TensorView
   const ThreadPredicateMap& thread_predicates_;
 
+  const ComputeAtRootDomainMap& ca_root_map_;
+
   IterDomainMap p2c_root_map_;
```

Comment:
Having two root mappings may not be necessary. I'll look into this later (#495).